ludolph/bot.py

```diff
@@ -671,7 +682,7 @@ def get_jid(self, msg, bare=True):
             jid = msg['from']
 
         if bare and jid:
-            return jid.bare
+            return self._sleekxmpp_fix_jid(jid).bare
```

Comment:
by default we require bare jid when calling `get_jid` however in this function we dont know if it is groupchat or regular chat when calling this function. Eg. in certain cases we got an exception due to change in SleekXMPP, therefore I am sending jid to `_sleekxmpp_fix_jid` so it doesnt matter if there is string or object it allways return JID object.

other affected place was `_get_room_member` when Ludolph was joining the room and checking who is in. We call this function there as well...